xdotool/functions

```diff
@@ -55,14 +55,24 @@ function zsh-notify() {
 
     zstyle -s ':notify:' "$type"-sound notification_sound
     zstyle -s ':notify:' "$type"-icon icon
+    zstyle -s ':notify:' app-name app_name
+    zstyle -s ':notify:' expire-time e_time
 
     title=$(notification-title "$type" time_elapsed "$time_elapsed")
 
     if [[ -n "$icon" ]]; then
         icon_option="-i $icon"
     fi
 
-    notify-send ${=icon_option} "$title" "$message"
+    if [[ -n "$e_time" ]]; then
+        e_time_option="-t $e_time"
```

Comment:
maybe check if `notify-send` supports `-t 0` as a way of saying "no timeout", then you can just set that as the default value in `notify.plugin.zsh` and avoid the "if" here